tests/support.rs

```diff
@@ -0,0 +1,92 @@
+extern crate env_logger;
```

Comment:
So that's why the support module for `check` is in its own folder! I didn't know that, will fix!